app/models/support/requests/content_publisher_feedback_request.rb

```diff
@@ -0,0 +1,32 @@
+require 'active_support/core_ext'
+
+module Support
+  module Requests
```

Comment:
We seem to be missing the unit tests for this class

Reply:
Done. We now have unit tests for this request.